KWStyle.xml

```diff
@@ -0,0 +1,131 @@
+<Project>LMMS</Project>
```

Comment:
This is useless, I can just remove it if you want ?

Ok, I choosed this name because it was the one given as an example on the doc. Will do.